core/src/org/javarosa/xpath/XPathConditional.java

```diff
@@ -120,24 +120,15 @@ private static void getExprsTriggersAccumulator(XPathExpression expr,
             TreeReference ref = ((XPathPathExpr)expr).getReference();
             TreeReference contextualized = ref;
 
-            if (ref.getContext() == TreeReference.CONTEXT_ORIGINAL) {
-                // Starts with 'current()' so contextualize in terms of the
-                // nodeset's original reference.
+            if (ref.getContext() == TreeReference.CONTEXT_ORIGINAL ||
+                    (contextRef == null && !ref.isAbsolute())) {
+                // Expr's ref begins with 'current()' or is relative and the
+                // context ref is missing.

```

Comment:
Allows for the relative path to be correctly contextualized in \<bind nodeset="/data/a/b/c" relevant="count(../../b[@id = 'two']) = 1" .../>